processor/transformprocessor/README.md

```diff
@@ -347,6 +348,49 @@ Examples:
 
 - `copy_metric(desc="new desc") where description == "old desc"`
 
+### aggregate_on_attributes
+
+`aggregate_on_attributes(function, Optional[attributeSet])`
```

Comment:
```suggestion
`aggregate_on_attributes(function, Optional[attributes])`
```
What do you think about removing `set` from the name? To me, `set` implies it may be a map.